Jenkinsfile

```diff
@@ -1103,5 +1142,18 @@ pipeline {
                 }
             }
         }
+        stage("Perf Test") {
```

Comment:
Please make the stage name comply with the documentation https://github.com/ROCmSoftwarePlatform/MIOpen/blob/b3c1ae6db390f55b449bb2e0247ef6e3a093290c/Jenkinsfile#L330-L350

Please extend the documentation if necessary.

Reply:
Resolved.